DOCS-765: Remove /instance refs and update example files link (#1005)
hazelcast/hz-docs: docs/modules/configuration/pages/jet-configuration.adoc

====

Note that, Hazelcast's Jet engine is disabled by default when you use Hazelcast

> within the Java context, i.e. `hazelcast.jar`. When using Hazelcast ZIP/TAR distributions

Reviewer: ```suggestion
within the Java context; that is, `hazelcast.jar`. When using Hazelcast ZIP/TAR distributions
```

== List of Configuration Options

> The _hazelcast-full-example_ configuration files (YAML, XML) available in your distribution and linked from xref:configuration:configuring-declaratively.adoc[] include a description of all configuration options. The configuration options for Jet engine are also listed below:

Reviewer: can we clarify that it isn't that both YAML and XML config files need to be updated?

Author: Not sure what you mean here - the files are included just for reference. For context I changed the hardcoded link to these files to point to the page instead - are you meaning we should provide more context on how these files are used?

Author: Op, took your suggestion - think that clarifies

> The _hazelcast-full-example_ configuration files (YAML, XML) available in your distribution and linked from xref:configuration:configuring-declaratively.adoc[] include a description of all configuration options. The configuration options for Jet engine are also listed below:

Reviewer: ```suggestion
The _hazelcast-full-example_ configuration files (available as YAML or XML) available in your distribution and linked from xref:configuration:configuring-declaratively.adoc[] include a description of all configuration options. The configuration options for Jet engine are also listed below:
```

Alternatively, you can add `hazelcast-client.yaml/xml` to the classpath or
working directory which will be picked up automatically. The location of
the file can also be given using the `hazelcast.client.config` system

> property, i.e. `-Dhazelcast.client.config=C:/myhazelcast-client.yaml/xml`.

Reviewer: ```suggestion
property; that is, `-Dhazelcast.client.config=C:/myhazelcast-client.yaml/xml`.
```
